packaging/crio.conf.d/10-microshift_amd64.conf

```diff
@@ -1,3 +1,10 @@
+[crio]
+# the presence of a version_file path will be result in
+# only the wipe of containers, only. Image deletion is
+# controlled by version_file_persist, which is unset by
+# default and thus will not cause images to be deleted.
+version_file="/var/run/crio/version"
```

Comment:
I tried to test this, but it does not seem to work as we want it. Here's the sequence I used.
* Install and enable crio + crio-wipe
* Enable version_file option
* Install and start microshift (not enable)
* Wait until all pods are up and running
* Run `sudo reboot` - normal reboot
* When the system is up, run `crictl images` and `crictl pods` - images are there, but all pods get deleted
* Start (not enable) microshift again 
* Wait until all pods are recreated
* Run `sudo reboot -f` - failure simulation
* When the system is up, run `crictl images` and `crictl pods` - **both pods and images are gone**

We never want to lose images...